CLAUDE.md

```
@@ -12,17 +20,22 @@
 - **Security audit**: `cargo audit`
 - **Benchmarking**: `cargo bench`
 - **Code coverage**: `cargo tarpaulin`
-- **Run with MCP inspector**: `npx @modelcontextprotocol/inspector cargo run`
+
+## OAuth Authentication
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)